pages/blog/migrating-from-rest.mdx

```diff
@@ -0,0 +1,133 @@
+import withArticle from '../../ui/blog/article';
+
+export const meta = {
+  title: 'Migrating from Schemaless REST API to GraphQL',
+  tags: ['graphql', 'rest', 'graphql-mesh'],
+  author: 'arda',
+  date: '2020-05-31',
+  description:
+    "It is really easy to migrate to GraphQL from a REST API without writing everything from scratch using GraphQL Mesh even you don't have any schema for your existing REST API.",
+  image:
+    '/blog-assets/migrating-from-rest/cover.png',
+};
+
+export default withArticle({ ...meta });
+
+Migrating from REST API to GraphQL may look trivial if you want to avoid rewriting your backend from scratch because your existing REST API might not have a decent schema even if you decide to use GraphQL as a proxy. In that case you still need to write a full schema for yout all endpoints.
+
+### Implementing a basic proxy GraphQL backend
+
+Let's say you have `/user` endpoint that does all CRUD operations for `User` entity with different HTTP methods, and you would need a GraphQL schema like below;
+
+```graphql
+type Query {
+  user(id: ID): User
+}
+
+type Mutation {
+  createUser(input: UserInput): User
+  updateUser(id: ID, input: UserInput): User
+  deleteUser(id: ID): ID
+}
+
+type User {
+  id: ID
+  name: String
+  age: Int
+}
+
+input UserInput {
+  name: String
+  age: Int
+}
+```
+
+And you would also need a thin business logic that proxies upcoming GraphQL requests to the REST API using GraphQL resolvers like below;
+
+```js
+module.exports = {
+   Query: {
+      user: (root, args) => fetch('https://myrest.com/user/' + args.id)
+      .then(res => res.json())
+   },
+  Mutation: {
+     createUser: (root, args) => fetch('https://myrest.com/user', {
+       method: 'PUT',
+       body: JSON.stringify(args.input)
+      }).then(res => res.json()),
+    updateUser:  (root, args) => fetch('https://myrest.com/user' + args.id, {
+       method: 'POST',
+       body: JSON.stringify(args.input)
+    }).then(res => res.json()),
+    deleteUser:  (root, args) => fetch('https://myrest.com/user' + args.id, {
+       method: 'DELETE'
+    }).then(res => res.json()),
+};
+```
+
+This example assumes that you have `/user/:id` endpoint that gets a `User` entity with `HTTP GET`, deletes user with `HTTP DELETE` and updates a `User` that has the given id with the given input. Also `/user` endpoint creates a new `User` with the given input.
+
+But this implementation will be hard to maintain when the REST API is updated and become bigger.
+
+### Using GraphQL Mesh instead without any code
+
+GraphQL Mesh is a tool that handles multiple non-GraphQL data sources and generates an executable GraphQL schema on top of them with a simple configuration file.
+
+It has JSON Schema handler that generates a GraphQL Schema based on the given JSON schema files. This handler can also generate JSON Schema on runtime based on the given sample request and response data.
```

Comment:
Add this to the begging of the sentence: 

On top of having handlers that automatically take care of sources with schema like - OpenAPI/Swagger, gRPC, SOAP, and others, it also has .......

also add links to each handler mentioned above and "the others" a link to docs